frontend/css/sidebar.less

```diff
@@ -74,6 +74,10 @@
   }
 }
 
+.sidebar-fresh-releases {
+  max-height: 85vh;
+}
+
```

Comment:
Since this css is only relevant for fresh-releases page, you can consider moving it to [that page's css file](https://github.com/suvid-singhal/listenbrainz-server/blob/3734db6c52ad621c16f1dd01346ae4cb420a65f1/frontend/css/fresh-releases.less)

Reply:
Done :)